atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleKeyValueServiceConfig.java

```diff
@@ -30,6 +30,16 @@
 
     public abstract JdbcHandler jdbcHandler();
 
+    @Value.Default
+    public String singleOverflowTable() {
+        return "atlas_overflow";
+    }
+
+    @Value.Default
+    public String overflowTablePrefix() {
+        return "ao_";
```

Comment:
Since there is no oracle driver there is setup that actually works so this is kind of only a theoretical question.  But if theoretical you were able to start up an oracle instance you would have to use either the overflow table prefix or the single overflow table.  Better would be to make a new option to use native support and then use neither of these.  Basically I don't see a world where we let open source users use this until we clean this all up a bunch (overflow table clean up is already on the cruft ticket).